polynote-frontend/polynote/ui/component/notebook/notebookconfig.ts

```diff
@@ -351,6 +351,24 @@ class Resolvers extends Disposable {
             } else { return [] }
         });
     }
+
+    static parseWrappedResolvers(wrappedResolvers: any): RepositoryConfig[] {
+        return wrappedResolvers.map((wrappedResolver: any) => {
```

Comment:
Same as above - can we narrow the `any` to something more specific?

Reply:
not sure - technically the JSON.parse call in the pasteConfig method _thinks_ it's parsing paste.repositories to a RepositoryConfig object (which it isn't, it's the wrapped anon object). the objects inside the array can be that same {type: "ivy" | ...etc } object, but not sure how to get the argument to be more specific without casting paste.repositories to `unknown` first (maybe that's okay?).